cli_reference/developer_cli_odo/creating-instances-of-services-managed-by-the-operators.adoc

```diff
@@ -0,0 +1,27 @@
+[id=creating-instances-of-services-managed-by-the-operators]
+= Creating instances of services managed by the Operators
```

Comment:
As written, the title is a bit awkward. Maybe something along the lines of, "Creating instances of services managed by Operators" could work?